bin/terminus

```diff
@@ -34,6 +34,7 @@ $output = new ConsoleOutput();
 
 // Create a config object.
 $config = new DefaultsConfig();
+$config->extend(new YamlConfig($config->get('user_home') . '/.terminus/config.yml'));
```

Comment:
It might not be clear from the word 'extend', but later configs override earlier configs in this chain. That means the built in constants will always override the user specified `config.yml` which is probably not what you wanted. 

I would say the order we want for this is between the shipped defaults and the env var overrides.